actor/inbox_test.go

```
@@ -31,6 +31,33 @@ func TestInboxSendAndProcess(t *testing.T) {
+		msg := Envelope{}
+		inbox.Send(msg)
+
+		timer := time.NewTimer(time.Second)
+		select {
+		case <-processedMessages: // Message processed
+		case <-timer.C:
+			t.Errorf("Message was not processed in time")
+		}
+		timer.Stop()
+
+		inbox.Stop()
```

golangci-lint (1.62.2)

Error return value of `inbox.Stop` is not checked

(errcheck)